modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -153,56 +157,123 @@ def partitioned_file(
             Indicate quote in a file.
         is_quoting: bool, default True
             Whether or not to consider quotes.
+        header_size: int, default 0
+            Number of rows, that occupied by header.
+        pre_reading: int, default 0
+            Number of rows between header and skipped rows, that should be read.
+        encoding: str, default None
+            `encoding` parameter of read_* function.
 
         Returns
         -------
-        An array, where each element of array is a tuple of two ints:
-        beginning and the end offsets of the current chunk.
+        List of tuples with the next elements:
+            int: partition start read byte
+            int: partition end read byte
+            int, array-like or callable: skiprows object aligned (adopted)
+                to the exact partition
+            int: number of rows that exact partition will contain
         """
+        read_rows_counter = 0
+        considered_rows_counter = 0
+        outside_quotes = True
+        if encoding is not None:
+            partition_skiprows = 1
```

Comment:
`partition_skiprows` already defined in `modin/engines/base/io/text/csv_dispatcher.py`(117 120 line) Why are you doing this again?

Reply:
Right, definition in `modin/engines/base/io/text/csv_dispatcher.py` is excessive, so it was removed.